src/Pages/Home.js

```diff
@@ -6,7 +6,21 @@ function Home(){
         <div className="Home">
             {/* Hero section container  */}
             <div className="Hero_section">
-                <img className="Hero_image" src={require('../Assets/HeroImg.jpg')} />
+            <img className="Hero_image" src= {require('../Assets/HeroImg.jpg')} />
+                <div className="Hero_overlay">
+                    <p className="style_2">Hello, my name is</p>
```

Comment:
className naming should always be consistent. Either always use camelCase or always use PascalCase.